splinterd/src/config/error.rs

```diff
@@ -21,12 +23,17 @@ use toml::de::Error as TomlError;
 #[derive(Debug)]
 /// General error type used during `Config` contruction.
 pub enum ConfigError {
-    ReadError { file: String, err: io::Error },
+    ReadError {
```

Comment:
This change is not mentioned in the commit message - could be a separate commit, or was added by accident.

Reply:
Ahh yeah rustfmt  did it for me, Ill pull it out